# Barcode Sequencing
by Hector L. Burgos

This protocol describes how to perform Barcode Sequencing (BarSeq) experiments to study *Vibrio fischeri* population dynamics when using mutant strains generated via the Sequence-Tagged-Gene-Deletion protocol published from our lab [Burgos et al (2020)](https://paperpile.com/shared/nhf9AU) and described in detail in the [Sequence-Tagged-Gene-Deletion protocol](sequence-tagged-gene-deletion.md). In summary, strains generated by this protocol will have inserted into their chromosome a scar containing a unique barcode sequence that identifies each individual strain. Mixed populations of barcoded strains can be assayed via amplicon PCR using oligos that anneal universal sequences within the barcoded scars and sequenced via Illumina sequencing platforms. These strains are versatile for probing competitive phenotypes and can be used in either *in vitro* and/or squid colonization experiments.

## I - Setting up the multiplexed population of barcode-tagged *Vibrio fischeri* strains

1. Inoculate 3 mL of LBS with the desired barcode-tagged strains and grow overnight at 25°C with aeration.

###### Next day

2. Mix 2 mL of each stationary phase culture in a 50 mL conical tube on ice.
3. Vortex well, place back on ice.
4. Spin down cells in swinging bucket rotor @ 4,700 rpm for 15 min and 4°C, then quickly but gently discard supernatant.
5. Resuspend cells in 1/10 the original volume of LBS (e.g., for 20 strains: original volume = 2 mL * 20 strains = 40 mL. 1/10 = 4 mL LBS).
6. Transfer 1.2 mL of the multiplexed population into a red-cap cryovial with 600 µl 50% glycerol (16.7% glycerol final).
7. Mix well by fully inverting vial 20 times, then freeze @ -80°C.

### Day 1 or 2
